source/databases-collections.txt

```diff
@@ -0,0 +1,275 @@
+You can also control options for how the driver waits for acknowledgment of
+read and write operations on a replica set by setting a **read concern** and a
+**write concern**.
+
+By default, databases inherit these settings from the ``MongoClient`` instance,
+and collections inherit them from the database. However, you can change these
+settings on your database or collection by using one of the following methods:
+
+- ``get_database()``: Gets the database with its settings inherited
+  from the client and changes the specified settings.
+- ``database.with_options()``: Gets the database and changes the specified
+  settings.
+- ``get_collection()``: Gets the collection and changes the specified settings.
+- ``collection.with_options()``: Gets the collection with its settings inherited
+  from its database and changes the specified settings.
+
+To change read or write settings with the preceding methods, call the method and
+pass in the collection or database name, and the new read preference, read
+concern, or write preference.
+
+The following example shows how to change the read preference, read concern and
+write preference of a database called ``test-database`` with the ``get_database()`` method:
+
+.. code-block:: python
+
+   client.get_database("test-database", 
+                       read_preference=ReadPreference.SECONDARY,
+                       read_concern="local",
+                       write_concern="majority")
+
+The following example shows how to change read and write settings of a
+collection called ``test-collection`` with the ``get_collection()`` method:
+
+.. code-block:: python
+
+   database.get_collection("test-collection", 
+                       read_preference=ReadPreference.SECONDARY,
+                       read_concern="local",
+                       write_concern="majority")
+
+The following example shows how to change read and write settings of a
+collection called ``test-collection`` with the ``with_options()`` method:
+
+.. code-block:: python
+
+   collection.with_options(read_preference=ReadPreference.SECONDARY,
+                           read_concern="local",
+                           write_concern="majority")
+
+.. tip::
+
+   To see the types of read preferences available in the ``ReadPreference`` enum, see the
+   `API documentation <{+api-root+}pymongo/read_preferences.html#pymongo.read_preferences.ReadPreference>`__.
+
+To learn more about the read and write settings, see the following guides in the
+MongoDB Server manual:
+
+- :manual:`Read Preference </core/read-preference/>`
+- :manual:`Read Concern </reference/read-concern/>`
+- :manual:`Write Concern </reference/write-concern/>`
+
+Tag Sets
+~~~~~~~~
+
+You can :manual:`tag </core/read-preference-tags/>` replica-set
+members according to any criteria you choose. By default, {+driver-short+} ignores tags
+when choosing a member to read from, but you can configure your read preference with
+a ``tag_sets`` parameter.
```

Comment:
This was a little confusing. Maybe break it up:

```suggestion
when choosing a member to read from. To instruct {+driver-short+}
to prefer certain tags, pass the ``tag_sets`` parameter to the
``_ServerMode`` constructor.
```

[_ServerMode](https://github.com/mongodb/mongo-python-driver/blob/167b9648caf94d57ee618ed240d7910f4553b55e/pymongo/read_preferences.py#L105)

Reply:
Changed the wording here slightly since I think it was a little bit off originally